packages/api/core/src/util/forge-config.ts

```diff
@@ -125,8 +127,13 @@ export default async (dir: string): Promise<ResolvedForgeConfig> => {
   if (await forgeConfigIsValidFilePath(dir, forgeConfig)) {
     try {
       // The loaded "config" could potentially be a static forge config, ESM module or async function
-      // eslint-disable-next-line @typescript-eslint/no-var-requires
-      const loaded = require(path.resolve(dir, forgeConfig as string)) as MaybeESM<ForgeConfig | AsyncForgeConfigGenerator>;
+      let loaded;
+      try {
+        loaded = (await dynamicImport(path.resolve(dir, forgeConfig as string))) as MaybeESM<ForgeConfig | AsyncForgeConfigGenerator>;
```

Comment:
Can we dedupe the `path.resolve` used in both try and catch into a local variable

Reply:
Done!